[docs] update changelog for release-xoai final (#476)
mapbox/mapbox-gl-native-android: CHANGELOG.md

 - Clarified the error message when an HTTP request has been cancelled. ([#433](https://github.com/mapbox/mapbox-gl-native-android/pull/433))
 - Fixed a memory leak that would occur when a source was removed. ([#412](https://github.com/mapbox/mapbox-gl-native-android/pull/412))
 - Fixed a bug that would halt the application during a scheduled snapshot when the map renderer was stopped. ([#390](https://github.com/mapbox/mapbox-gl-native-android/pull/390)

>  - Not change file source status while changing cache path ([#457](https://github.com/mapbox/mapbox-gl-native-android/pull/457))

Reviewer: ```suggestion
 - Avoid changing file source status when the cache path is changed. ([#457](https://github.com/mapbox/mapbox-gl-native-android/pull/457))
```

>  - Avoid null pointer exception when the underlying hardware is too slow to create the surface ([#471](https://github.com/mapbox/mapbox-gl-native-android/pull/471))

Reviewer: ```suggestion
 - Avoid null pointer exception when the underlying hardware is too slow to create the surface. ([#471](https://github.com/mapbox/mapbox-gl-native-android/pull/471))
```

### User location indicator
 - Double-tapping the map when the camera is tracking the user location indicator now zooms to the `LocationComponent`'s location. ([#378](https://github.com/mapbox/mapbox-gl-native-android/pull/378))
 - Improved camera rotation to use the shortest path. ([#426](https://github.com/mapbox/mapbox-gl-native-android/pull/426))
 - Added support for using a background image on GPS mode when `LocationComponentActivationOptions#useSpecializedLocationLayer` is activated . ([#439](https://github.com/mapbox/mapbox-gl-native-android/pull/439))
 - Fixed a bug where an in-progress animation of an updating `LocationComponent` would be interrupted when a style was changed. ([#437](https://github.com/mapbox/mapbox-gl-native-android/pull/437))
 - Updated documentation to reflect that the `FOREGROUND_LAYER` constant is usable for relative positioning in both normal and specialized rendering modes. [#413](https://github.com/mapbox/mapbox-gl-native-android/pull/413)

>  - Set gps bearing immediately while changing to gps mode. ([#470](https://github.com/mapbox/mapbox-gl-native-android/pull/470))

Reviewer: ```suggestion
 - Set GPS bearing immediately when changing the indicator to GPS mode. ([#470](https://github.com/mapbox/mapbox-gl-native-android/pull/470))
```
